_posts/2021-04-05-fmf-unified-testing.md

```diff
@@ -0,0 +1,235 @@
+---
+title: Unified upstream and downstream testing with TMT and Packit
```

Comment:
```suggestion
title: Unified upstream and downstream testing with tmt and Packit
```

Hi, I would propose to use `tmt` always in lower case. It's a command line tool and thus I'd say this spelling seems to be more suitable. Just my two cents...

Reply:
Will do that, thanks! I'll replace the other instances as well. @garrett still has some fixes incoming, once they arrive I'll push this "tmt" change too.

Reply:
Great, thanks!

Reply:
FWIW: Fedora's docs @ https://docs.fedoraproject.org/en-US/ci/tmt/ have "Test Management Tool" when talking about it as a concept, but uses lowercase for `tmt` when it's talked about as a tool. Same for Packit and some other tooling.

Some projects have a split between the name of the project or concept like this and their tool. Others, such as darktable, are always stylistically lowercased. I was referring to various docs to try to get the usage right here. :shrug: But it's hard, apologies for getting it wrong.

So we should be consistent in the title. If we're talking about the tools, then lowercase — probably for both. If the projects, then Title Case (or uppercase for acronyms of the project)? And we should be consistent in the document outside of the title too.

And I guess if Fedora's docs have it wrong as well, then they should also be fixed. (Outside of this PR of course. :wink:)

Reply:
Yeah, naming is hard :) For `tmt` I tend to you use just two forms:

* `Test Management Tool` usually for headings or when explaining the abbreviation (used quite rarely)
* `tmt` for all other occurrences (almost everywhere)

This should also be the case for both [fedora](https://docs.fedoraproject.org/en-US/ci/tmt/) and [upstream](https://tmt.readthedocs.io/en/stable/) docs.

Reply:
Replaced everything with "tmt", thanks!